### PR TITLE
tab ui

### PR DESCRIPTION
**Pull request by Prakkkmak**

### CODE REVIEW

**Comment by Prakkkmak** [comment]
[Tab with list and round infos](https://trello.com/c/WjuEcSef/40-tab-with-list-and-round-infos)

File: code/Gamemode.cs [hunk 1]
```diff
@@ -20,6 +20,8 @@ public Gamemode()
 
         public static Gamemode Instance { get; private set; }
 
+        public event Action<MainPlayer> PlayerAddedEvent;
```
Comment (Nowp) [inline code/Gamemode.cs:23]: Is it not EvenHandler we should use ?

Comment (Prakkkmak) [reply code/Gamemode.cs:23]: What event handler ?

Comment (Nowp) [reply code/Gamemode.cs:23]: Event is designed with EventHandler in mind

https://stackoverflow.com/questions/1431359/event-action-vs-event-eventhandler
https://docs.microsoft.com/fr-fr/dotnet/api/system.eventhandler?view=net-5.0

Comment (Prakkkmak) [reply code/Gamemode.cs:23]: I wanted to follow sbox coding with their event. But I didn't know about eventhandler ! Do you want me to change ?

Comment (Nowp) [reply code/Gamemode.cs:23]: Yep, the convention seems to be EventHandler. I don't know why facepunch didn't follow it lmao

**Review by Nowp — APPROVED** [review]

**Pull request merged**